apis/Google.Cloud.Spanner.Data/docs/connection_string.md

```diff
@@ -134,3 +134,54 @@ documentation.
 Example:
 
 - EmulatorDetection=EmulatorOrProduction
+
+## ClrToSpannerTypeDefaultMappings
+
+ Option to configure the default CLR type to `SpannerDbType` mapping. 
+ This option comes into picture only if `SpannerDbType` and `DbType` of the `SpannerParameter`
+ are not explicitly provided. Currently only `decimal` and `DateTime` CLR types are supported.
+
+ The valid type mappings for `decimal` are:
+
+- DecimalToFloat64 - `decimal` CLR type will map to `SpannerDbType.Float64` if `SpannerDbType` 
+and `DbType` are not explicitly provided for the `SpannerParameter`.
+- DecimalToNumeric - `decimal` CLR type will map to `SpannerDbType.Numeric` if `SpannerDbType` 
+and `DbType` are not explicitly provided for the `SpannerParameter`. This should be used while 
+working with Google Standard SQL dialect only. 
+- DecimalToPgNumeric - `decimal` CLR type will map to `SpannerDbType.PgNumeric` if `SpannerDbType` 
+and `DbType` are not explicitly provided for the `SpannerParameter`. This should be used while
+working with PostgreSQL dialect only.
+
+The valid type mappings for `DateTime` are:
+
+- DateTimeToDate - `DateTime` CLR type will map to `SpannerDbType.Date` if `SpannerDbType` 
+and `DbType` are not explicitly provided for the `SpannerParameter`.
+- DateTimeToTimestamp - `DateTime` CLR type will map to `SpannerDbType.Timestamp` if `SpannerDbType` 
+and `DbType` are not explicitly provided for the `SpannerParameter`.
+
+The mapping can be provided as comma separated values. Only one mapping for a type must be provided. 
+Providing mutiple mapping for a type, or providing invalid mapping or providing whitespaces will result in `ArgumentException`.
+
+Examples:
+
+- ClrToSpannerTypeDefaultMappings=DecimalToFloat64,DateTimeToDate
+- ClrToSpannerTypeDefaultMappings=DecimalToNumeric,DateTimeToTimestamp
+
+## SpannerToClrTypeDefaultMappings
+
+Option to configure the default `SpannerDbType` to CLR type mappings. This option comes into picture 
+only if CLR type of the value being read is not explicitly provided while reading the data from the database.
+E.g., `ExecuteScalarAsync<object>()`. Currently only `SpannerDbType.Date` is supported.
```

Comment:
I'd put the example at the end of the previous sentence, and I'd probably use `reader["fieldName"]` as the example. (I think it's somewhat unlikely for people to *explicitly* specify a type argument of `object` - it's more likely that they'll use non-generic APIs such as the reader indexer.)